### ISSUE TITLE
'cask exec' crashes

### ISSUE THREAD

**Issue by Wilfred** [cask/cask#374]
```
$ cask exec
Traceback (most recent call last):
  File "/home/wilfred/.cask/bin/cask", line 347, in <module>
    main()
  File "/home/wilfred/.cask/bin/cask", line 328, in main
    exec_command(sys.argv[2:])
  File "/home/wilfred/.cask/bin/cask", line 278, in exec_command
    os.execvp(command[0], command)
IndexError: list index out of range
```

Perhaps we should just show the help message in this situation? Alternatively, a more friendly error message would be nice.

**Comment by lunaryorn** [comment]
@Wilfred I have no time to fix this; a pull request would be very welcome :blush:

**Comment by lunaryorn** [comment]
Should be fixed by #375.
